website/sidebars.json

```diff
@@ -145,7 +145,8 @@
       "api/quickstart"
     ],
     "JSON RPC": [
-      "interaction/rpc"
+      "interaction/rpc",
+      "api/rpc-params"
```

Comment:
I don't feel good about having an endless document of all the APIs, but having such a split does not make a lot of sense to me unless we choose to only cover the fact that we use block ID || finality, and migrate to an object-based API (instead of position-based).

Reply:
i was separating this as part of a preview (what's on stagenet, for example, vs testnet)